roles/systemd-docker-service/defaults/main.yml

```diff
@@ -16,3 +16,6 @@ systemd_service_after: docker.service
 systemd_service_bindsto:
 systemd_service_wants: docker.service
 systemd_docker_image_tag: latest
+
+systemd_docker_log_opt_max_file: 2
+systemd_docker_log_opt_max_size: 10m
```

Comment:
10m would be a huge file, why not sticking to 5x4m?

Reply:
sure!